erpnext/patches/v13_0/requeue_failed_reposts.py

```diff
@@ -0,0 +1,14 @@
+import frappe
+from frappe.utils import cstr
+
+def execute():
+
+	reposts = frappe.get_all("Repost Item Valuation",
+		{"status": "Failed", "modified": [">", "2021-10-6"] },
```

Comment:
```suggestion
		{"status": "Failed", "modified": [">", "2021-10-05"] },
```

dates should be zero-padded ideally so string comparison can also work on them.